include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -159,7 +159,7 @@ class MGTwoLevelTransfer
    * Perform prolongation.
    */
   void
-  prolongate(VectorType &dst, const VectorType &src) const;
+  prolongate_and_add(VectorType &dst, const VectorType &src) const;
```

Comment:
This technically represents a break in compatibility, but I think users will not often use these classes directly in their code. Do you think we need a changelog entry?

Reply:
I cannot image that someone uses `MGTwoLevelTransfer` in its raw fashion to prlongate/restrict. One would use `MGTransferGlobalCoarsening` for that purpose.

Reply:
This is also what I thought. Let's go ahead then.